react/features/base/util/uri.js

```diff
@@ -363,6 +363,9 @@ export function parseURIString(uri: ?string) {
     }
     obj.room = room;
 
+    // The part of the pathname from the beginning to the room name is the tenant.
+    obj.tenant = contextRootEndIndex > 1 ? pathname.substring(1, contextRootEndIndex) : '';
```

Comment:
Maybe extract it from the room? I think it's more obvious when reading...

Reply:
I don't think the room includes the tenant.